lib/grammers-tl-types/tl/api.tl

```diff
@@ -1890,9 +1978,9 @@ messages.getChats#49e9528f id:Vector<long> = messages.Chats;
 messages.getFullChat#aeb00b34 chat_id:long = messages.ChatFull;
 messages.editChatTitle#73783ffd chat_id:long title:string = Updates;
 messages.editChatPhoto#35ddd674 chat_id:long photo:InputChatPhoto = Updates;
-messages.addChatUser#f24753e3 chat_id:long user_id:InputUser fwd_limit:int = Updates;
+messages.addChatUser#cbc6d107 chat_id:long user_id:InputUser fwd_limit:int = messages.InvitedUsers;
```

Comment:
Damn, another instance of Telegram shoving updates behind things that aren't updates.

We will need to add this new type here:
https://github.com/Lonami/grammers/blob/aa8d52762d5680abbb2deeb6e639416c81d4714c/lib/grammers-mtproto/src/mtp/encrypted.rs#L28-L36

and handle it inside this `extend`:
https://github.com/Lonami/grammers/blob/aa8d52762d5680abbb2deeb6e639416c81d4714c/lib/grammers-mtsender/src/lib.rs#L561